webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -60,6 +60,12 @@ class SubscriptionsPage extends Component {
     if (numberOfTasks > 0) {
       if (numberOfPrevTasks === 0 || prevProps.tasks[0].id !== tasks[0].id) {
         [task] = this.props.tasks;
+        const taskStatedMessage = Jed.sprintf(__('%s task has started.'), task.humanized.action); 
```

Comment:
Is it possible to include a link to the task like we do with the other toast notifications?

Reply:
sure

Reply:
There's a typo in the variable name: 
```
taskStartedMessage
       ^
```